Systems/oil.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+
+<!--
+***********************************************************************************
+Extra weight due to oil quantity
+***********************************************************************************
+ -->
+
+<system name="oil">
```

Comment:
Maybe you should combine `Systems/oil.xml` and `Systems/carburator.xml`. You could name the file `c172p-engine.xml`.Just them the functions in different channels.